meetings/2024-10-02.md

```diff
@@ -0,0 +1,109 @@
+* RG: That's one measure and so many different ways of measuring. It is not something we can solve for, but to have an intuition for.
+* MdJ: If you look to see what solutions get traction, that is a proxy for how complex they are to implement. Look at shape files for instance, very few app devs write them, I think that's an indication that it's quite a complex solution.
+* SC: Use cases are important. In the past we had a use case survery, which ties to Michiel's point. This can signal from the community interest and commitment. People are brining up age old issues about Solid should be doing this or that. We have an aboundance of use cases. Data portability comes up, access requests. We had issues, we had proposals. For example [data migrator](https://pdsinterop.org/solid-migrator/). I'm also part of this and we didn't do great job on evaluating those proposals. We need storage provisiooning, migrating storate. There are proposals and we don't give them enough attention. Later in the future we come back "oh we need that". We do need those use cases but there are solutions in front of us. A solution is better than no solution. Recently we heard need for digital wallets, sometimes it is hard to dig out those proposals. We need to be evaluating those solutions, if we don't them enough attention people get impression that we don't take them seriously. I don't want us to get lost in the process and loos track of simple solutions in front of us.
+* eP: I hear there is an agreement on evaluating proposal. Also agree with SC on signal from community and what to implement. If UCs are real-world, then if a UC is not mine but an alternative is like mine, then I can use those three requirements. We can look at more generic and specific level. (from chat: example - https://github.com/solid/authorization-panel/tree/main/proposals/evaluation)
+* eP: re MdJ, we need to have a little more signaling. How the dev chooses one of the alternative proposals. With a matrix the dev can say I also have those requirements so I'll try to implement or 80% is good enough. Other people can make educated guesses on what to use. We also need the matrix of which proposal would allow implementers to decide. Also to know what they like or don't like.
+* MdJ: Also need people to vote with their feet.
+* RG: SC, one of the things is that all of these discusssions are very buried. I try to stab at one or two of these things, but we really need some effort to bring all of those from GH. You have this incredible domain knowledge due to you having been here for years but most of us are newer. There has to be more visibility on those discussions.
+* PAC: To react to SC, taking into account previous work, i think it is very important. It might be very importnt in the LWS WG for each UC. But also maybe reference earlier work in order to see what the CG has done with the UCs. So, ack not at all starting from scratch.
```

Comment:
```suggestion
* RG: SC, one of the things is that all of these discusssions are very buried. I try to stab at one or two of these things, but we really need some effort to bring all of those from GH. You have this incredible domain knowledge, due to you having been here for years, but most of us are newer. There has to be more visibility on those discussions.
* PAC: To react to SC, I think it is very important to take previous work into account. It might be very important in the LWS WG for each UC. But also, maybe reference earlier work, to see what the CG has done with the UCs. So, not at all starting from scratch.
```